sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Release History
 
-## 7.0.0-beta.2 (Unreleased)
-
+## 7.0.0-beta.2 (2020-05-07)
+
+- Add support for receiving messages from specific sessions
+- Add support for processing messages from multiple sessions
+- Add missing schedule and cancel APIs in ServiceBusSenderClient
+- Add support to send a collection of messages at once
```

Comment:
```suggestion
- Add support to send a collection of messages at once without needing to create a `ServiceBusMessageBatch` first. This will throw an error/exception will the messages cannot fit as per batch size restrictions
```